### PR TITLE
Update 2.x.md

### PR DESCRIPTION
**Pull request by burtonjc**
connect https://github.com/waffleio/waffle.io-takeout/issues/181

### CODE REVIEW

File: releases/2.x.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Waffle Takeout 2.4
 
+## Waffle Takeout 2.4.4
+_Released 2016.08.01_
+
+_Supported GH:E versions: GH:E 2.2+_
+
+#### Features
+
+- More intuitive setup for GitHub Enterprise and GitHub.com integrations.
```
Comment (songawee) [inline releases/2.x.md]: I like this, but I guess technically it was out in 2.4.3?

File: releases/2.x.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+- New account page giving an overview of a users Waffle account.
```
Comment (songawee) [inline releases/2.x.md]: users -> user's

File: releases/2.x.md (hunk 1 above)
Comment (burtonjc) [inline releases/2.x.md]: should i scratch it?

**Comment by songawee** [comment]
ha, nice updates. Was just going to write a note about the GH:E not requiring it. 👍 🚢 :shipit:

File: releases/2.x.md (hunk 1 above)
Comment (songawee) [inline releases/2.x.md]: either that or retroactively add it to 2.4.3? Sorry, that was my bad for not adding it. here is probably fine too. I'm pretty sure no one will really complain.

**Pull request merged**